.travis.yml

```diff
@@ -1,7 +1,20 @@
+env:
+  global:
+    - NODE_ENV=test
+    - secret = authorshaven
+    - DATABASE_TEST=postgres://postgres:admin@localhost/users_test
```

Comment:
We talked about this and decided that this is the travis.yml script that we will be using
<img width="1440" alt="Screen Shot 2019-07-09 at 08 55 09" src="https://user-images.githubusercontent.com/41841554/60865900-4d8cad00-a227-11e9-8470-13856110058c.png">